packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -186,7 +181,7 @@
     margin-bottom: 4px;
   }
 
-  .mdc-text-field--box + &,
```

Comment:
I'm a little nervous about changes like this that just remove `--box` because whereas `--box` only ever applied to one variant, `mdc-text-field` applies to _every_ MDC text field (thus making the `--outlined` selector after this completely redundant). Are we sure this is what we want, or should there be exceptions to this?

We should check this for anything where `--box` specifically was removed.

Reply:
This was the only area where --box was removed and the .mdc-text-field part of it was left. This affects specifically the helper text that only the default and outlined variants have.